Sources/Networking/HTTPClient/HTTPRequestPath.swift

```diff
@@ -260,6 +268,9 @@ extension HTTPRequest.Path: HTTPRequestPath {
         case .postRedeemWebPurchase:
             return "post_redeem_web_purchase"
 
+        case .appHealthReport:
+            return "get_app_health_report"
+
```

Comment:
we can do it as a follow-up, but we might as well remove the other health API and instead bridge internally to this one

Reply:
Oh yeah 100%, this is the next PR that I will be doing, getting rid of the calls to existing health endpoint from the SDK 💪

Reply:
Logged it as a ticket: https://linear.app/revenuecat/issue/DX-366/clean-up-the-old-health-endpoint-calls-from-the-sdk